docs/en/getting-started/install.md

```diff
@@ -50,7 +50,7 @@ sudo rpm --import https://repo.clickhouse.tech/CLICKHOUSE-KEY.GPG
 sudo yum-config-manager --add-repo https://repo.clickhouse.tech/rpm/stable/x86_64
 ```
 
-If you want to use the most recent version, replace `stable` with `testing` (this is recommended for your testing environments). The `prestable` tag is sometimes available too.
+If you want to use the most recent version, replace `stable` with `testing` (this is recommended for your testing environments). `prestable` is sometimes also available.
```

Comment:
it's not clear what exactly does "sometimes" mean

Reply:
If you don't know, who knows?

Reply:
@BayoNet well, we just need to properly document the release process as we've discussed recently on email.